sandbox/preact-ts/panda.config.ts

```diff
@@ -1,6 +1,11 @@
 import { defineConfig } from '@pandacss/dev'
 
 export default defineConfig({
+  conditions: {
+    mq: '@media (min-width: 640px)',
+    custom: '&[data-custom]',
+    supportHover: ['@media (hover: hover) and (pointer: fine)', '@supports (display: grid)', '&:hover'],
+  },
```

Comment:
Let's move these to a dedicated test somewhere in packages/core so we can be more confident it works in the future.